### DISCUSSION TITLE
求助，有没有懂代码的大佬修一下这个脚本，非常感谢。

### DISCUSSION THREAD

**Discussion by archerzhoujj** [the1812/Bilibili-Evolved#5090] [综合讨论 / General]
这个单曲循环用起来很方便，但最近ui更新后用不了了
https://greasyfork.org/zh-CN/scripts/450915-b%E7%AB%99%E8%A7%86%E9%A2%91%E5%8D%95%E6%9B%B2%E5%BE%AA%E7%8E%AF

**Comment by the1812** [comment]
![Snipaste_2024-12-28_13-59-51](https://github.com/user-attachments/assets/1635b757-435b-4ad7-8c0c-266d078a59c3)
